sound/soc/sof/pm.c

```diff
@@ -436,7 +436,7 @@ int snd_sof_resume(struct device *dev)
 	struct snd_sof_dev *sdev = dev_get_drvdata(dev);
 	int ret;
 
-	if (sdev->s0_suspend) {
+	if (snd_sof_pcm_d0i3_suspend(sdev)) {
```

Comment:
@keyonjie forgot to change the name here? and below?

Reply:
oops, my bad, shouldn't ignore compiling test for each change.